draft-ietf-privacypass-batched-tokens.md

```diff
@@ -78,18 +78,18 @@ multiple HTTP requests, these ad-hoc solutions vary based on transport protocol
 support. In addition, in some cases, they cannot take advantage of cryptographic
 optimizations.
 
-The first variant of the issuance protocol builds upon the privately verifiable
-issuance protocol in {{RFC9578}} that uses VOPRF {{!OPRF=RFC9497}}, and allows
-for batched issuance of tokens and amortizes the cost of zero knowledge proofs.
-This allows clients to request more than one token at a time and for issuers to
-issue more than one token at a time. In effect, private batched issuance
-performance scales better than linearly.
-
-The second variant of the issuance protocol introduces a new Client-Issuer
-communication method, which allows for batched issuance of generic token
-types. This allows clients to request more than one token at a time and for
-issuers to issue more than one token at a time. This variant has no other effect
-than batching requests and responses and the issuance performance remains
+The first variant ({{amortized-batch}}) of the issuance protocol builds upon the
+privately verifiable issuance protocol in {{RFC9578}} that uses VOPRF
+{{!OPRF=RFC9497}}, and allows for batched issuance of tokens and amortizes the
+cost of zero knowledge proofs. This allows clients to request more than one
+token at a time and for issuers to issue more than one token at a time. In
+effect, private batched issuance performance scales better than linearly.
+
+The second variant ({{generic-batch}}) of the issuance protocol introduces a new
+Client-Issuer communication method, which allows for batched issuance of generic
+token types. This allows clients to request more than one token at a time and
+for issuers to issue more than one token at a time. This variant has no other
+effect than batching requests and responses and the issuance performance remains
```

Comment:
I confirm this addresses the [comment](https://mailarchive.ietf.org/arch/msg/privacy-pass/WnRzS-JUhWeMEod8N4zTFBfaMvs/) about having links to the section associated to the protocol